hummingbot/connector/exchange/btc_markets/btc_markets_constants.py

```diff
@@ -0,0 +1,84 @@
+# A single source of truth for constant variables related to the exchange
+# https://api.btcmarkets.net/doc/v3#section/General-notes
+
+from hummingbot.core.api_throttler.data_types import RateLimit
+from hummingbot.core.data_type.in_flight_order import OrderState
+
+EXCHANGE_NAME = "btc_markets"
+DEFAULT_DOMAIN = "btc_markets"
+# Base URL
+REST_URLS = {"btc_markets": "https://api.btcmarkets.net/",
+             "btc_markets_test": "https://api.btcm.ngin.io/"}
```

Comment:
Out of curiosity where did you get the test endpoints from, I couldn't find any publicly available information on these or how to create accounts etc.

Reply:
Hi @lt7 , 

I work as an engineer at BTC Markets so I have access to this environment which I used to test the connector against during development. :) 

Regards,